### DISCUSSION TITLE
Docs->Deep Gaussian Process, Why output dimension is 2?

### DISCUSSION THREAD

**Discussion by findoctorlin** [cornellius-gp/gpytorch#2128] [Q&A] [answered]
Hello!

I am a student working on my master thesis related to Gaussian Process. I have read the paper for this tutorial https://arxiv.org/abs/1705.08933, which using the method of doubly stochastic variational GP for prediction.

In the tutorial it quite confused me why the output dimension is set for 2 instead for 1? 
![image](https://user-images.githubusercontent.com/57867183/189202007-a6249938-71e1-480c-aa4a-b43cb863b0f7.png)

Since in the paper the output dimension is 1 and also in the dataset we applied in this paper is also 1.
![image](https://user-images.githubusercontent.com/57867183/189202093-cfd0cd4a-2dcd-4a62-82e7-fa1659bd72e2.png)

Thank you!

**Comment by findoctorlin** [comment]
*The tutorial link is https://docs.gpytorch.ai/en/stable/examples/05_Deep_Gaussian_Processes/Deep_Gaussian_Processes.html

**Comment by gpleiss** [comment, marked as answer]
`output_dims` here refers to the number of outputs of the **first** (hidden layer).

Perhaps changing the variable name to `hidden_dims` would be clearer. Please open up a PR if you think that would be a useful change 😄
